crates/ruff/src/autofix/codemods.rs

```diff
@@ -11,119 +11,6 @@ use ruff_python_ast::source_code::{Locator, Stylist};
 use crate::cst::helpers::compose_module_path;
 use crate::cst::matchers::match_statement;
 
-/// Given an import statement, remove any imports that are specified in the `imports` iterator.
-///
-/// Returns `Ok(None)` if the statement is empty after removing the imports.
-pub(crate) fn remove_imports<'a>(
```

Comment:
Does this just need a rebase? Why was this deleted?

Reply:
i think the rebase went wrong, i have to fix that